babel/src/main/java/org/apache/calcite/sql/babel/SqlDateTimeAtTimeZone.java

```diff
@@ -30,32 +29,46 @@
 
 import java.util.List;
 
+/**
+ * A {@code SqlDateTimeAtTimeZone} is an AST node that describes
+ * the date time expression of At Time Zone.
+ */
 public class SqlDateTimeAtTimeZone extends SqlCall implements SqlExecutableStatement {
   public static final SqlSpecialOperator OPERATOR =
       new SqlSpecialOperator("AT TIME ZONE", SqlKind.OTHER);
 
   private final SqlNode dateTimePrimary;
-  private final SqlIdentifier timeZoneValue;
+  private final SqlNode displacementValue;
 
+  /**
+   * Creates a {@code SqlTranslateUsingCharacterSet}.
+   *
+   * @param pos  Parser position, must not be null
+   * @param dateTimePrimary  SqlNode, contains the time to be transformed
+   * @param displacementValue SqlNode, contains the Displacement to the time
```

Comment:
nit: /Displacement/displacement/

Reply:
Fix in https://github.com/googleinterns/calcite/pull/91/commits/10de7c276e756340204cb4dc21cc7f497fbdf157